<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewOrImportWalletWidget</class>
 <widget class="QWidget" name="NewOrImportWalletWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="newWalletBtn">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>245</y>
     <width>135</width>
     <height>34</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>创建钱包</string>
   </property>
  </widget>
  <widget class="QToolButton" name="importWalletBtn">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>290</y>
     <width>135</width>
     <height>34</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>导入钱包</string>
   </property>
  </widget>
  <widget class="QToolButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_version">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>114</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14px;</string>
   </property>
   <property name="text">
    <string>v1.0.1</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
